modules/ding_ting_frontend/ding_ting_frontend.module

```diff
@@ -64,3 +64,13 @@ function ding_ting_frontend_ding_entity_view($entity, $view_mode) {
   $entity->content['#groups']['group_holdings_available']->format_settings['formatter'] = $modified_view_mode;
 
 }
+
+/**
+ * Implements hook_post_features_revert().
+ */
+function ding_ting_frontend_post_features_revert($component) {
```

Comment:
On my environment it works. I've have reverted features a lot of time and can see that it still keep going normally.